src/Pkg.jl

```diff
@@ -264,6 +261,47 @@ const develop = API.develop
 #TODO: Will probably be deprecated for something in PkgDev
 const generate = API.generate
 
+"""
+    Pkg.info(; keywords...)::Dict{UUID, PackageInfo}
+    Pkg.info(uuid::UUID; keywords...)::Dict{UUID, PackageInfo}
+    Pkg.info(uuids::Vector{UUID}; keywords...)::Dict{UUID, PackageInfo}
+
```

Comment:
Document what the uuid is the uuid of.
I am pretty sure it is a package UUID.

Also you probably want to include instructions somewhere on how to retrieve a list of UUIDs for a given package name and a registry / locally.
(I feel like that is `pkgid`?)